### PR TITLE
Euclideon logo watermark option

### PR DESCRIPTION
**Pull request by DamianMadden**
Resolves [AB#335](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/335)

### CODE REVIEW

File: src/vcInternalTexturesData.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
...0x17, 0xcb, 0x18, 0x41, 0x10, 0xfc, 0x2e, 0x5a, 0x3f, 0xd5, 0xb5, 0x2b, 0xb1, 0x8c, 0x11, 0x04, 0xc1, 0xd7, 0xa2, 0xf5, 0xdf, 0xa8, 0xa9, 0x37, 0x79, 0x49, 0x5a, 0x41, 0x10, 0x66, 0x03, 0xaf, 0x86, 0x3c, 0xec, 0x03, 0xae, 0x04, 0xce, 0x17, 0xc1, 0x12, 0x04, 0xc1, 0xcf, 0x35, 0xad, 0x31, 0xcb, 0x98, 0xaf, 0x13, 0x8f, 0xf5, 0x4a, 0x72, 0x0a, 0x82, 0xe0, 0x67, 0xd1, 0xfa, 0x19, 0xca, 0x32, 0xe6, 0x19, 0x49, 0x46, 0x41, 0x10, 0xfc, 0x2c, 0x5a, 0x8f, 0xa2, 0xfa, 0xad, 0xee, 0x97, 0xe4, 0x13, 0x04, 0xa1, 0xd6, 0x54, 0xd3, 0xa7, 0xf5, 0x3c, 0x2f, 0x5a, 0xc6, 0x88, 0x60, 0x09, 0x82, 0xe0, 0xdb, 0x9a, 0x56, 0x11, 0x65, 0x19, 0xf3, 0x55, 0xb1, 0x8c, 0x11, 0x04, 0xc1, 0xef, 0xa2, 0x75, 0x2f, 0x6a, 0xb5, 0x66, 0xb1, 0x8c, 0x11, 0x04, 0xc1, 0xd7, 0xa2, 0xb5, 0x1d, 0x65, 0x19, 0xf3, 0x1f, 0x92, 0x44, 0x82, 0x20, 0xf8, 0x89, 0x89, 0x7d, 0x5a, 0x3d, 0xc0, 0x67, 0x80, 0x95, 0x22, 0x58, 0x82, 0x20, 0xf8, 0xb9, 0xa6, 0xe5, 0x00, 0xff, 0x08, 0x7c, 0x41, 0x2c, 0x63, 0x04, 0x41, 0xf0, 0xbb, 0x68, 0xfd, 0x06, 0x35, 0x84, 0x61, 0xb3, 0x24, 0x87, 0x20, 0x08, 0x7e, 0xe7, 0xff, 0x0f, 0x00, 0x2d, 0x3b, 0x36, 0xec, 0x36, 0x31, 0x0e, 0x0c, 0x00, 0x00, 0x00, 0x00, 0x49, 0x45, 0x4e, 0x44, 0xae, 0x42, 0x60, 0x82 }
```
Comment (samsinsane) [inline src/vcInternalTexturesData.h]: Can I veto this awful practice of jamming multiple file blobs into a single header?

Comment (DamianMadden) [reply src/vcInternalTexturesData.h]: You'd prefer single file blobs in a header? Paul specified on AB to embed the file so it's not easy to change

Comment (samsinsane) [reply src/vcInternalTexturesData.h]: > You'd prefer single file blobs in a header?

No, data doesn't need to be in headers and thus I'd prefer it if the data wasn't in the header.

> Paul specified on AB to embed the file so it's not easy to change

I know, but that doesn't require the data to be in the header.

Comment (OsiJr) [reply src/vcInternalTexturesData.h]: The other blobs are in `.cpp` files with externs in the header

Comment (samsinsane) [reply src/vcInternalTexturesData.h]: Not the `vcInternalModelsData.h`, the `vcInternalModels.h`/`.cpp` do but I think they're loaded not compiled in.

**Pull request merged**